src/rocm_docs/rocm_docs_theme/theme.conf

```diff
@@ -10,5 +10,5 @@ flavor = rocm
 
 link_main_doc = True
 
-header_latest_version = 5.6.1
-header_release_candidate = 5.7
+header_latest_version = 5.7
+header_release_candidate = 5.8
```

Comment:
6.0

Reply:
Fixed in [de84f5d](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/381/commits/de84f5db304b27fd4415df39ad80df4f25c33c10)